runtime/integration-tests/src/generic/cases/liquidity_pools.rs

```diff
@@ -5730,7 +5745,11 @@ mod altair {
 			env: &mut FudgeEnv<T>,
 			transfer_amount: Balance,
 			currency_id: CurrencyId,
-			meta: AssetMetadata<Balance, CustomMetadata>,
+			meta: AssetMetadata<
```

Comment:
You can use `cfg_types::tokens::AssetMetadata` which already has the expected bounds